File: rest/conference/instance-get-example-1/instance-get-example-1.js
// Download the Node helper library from twilio.com/docs/node/install
// These vars are your accountSid and authToken from twilio.com/user/account
var accountSid = '{{ account_sid }}';
var authToken = "{{ auth_token }}";
var client = require('twilio')(accountSid, authToken);

client.conferences("CFbbe46ff1274e283f7e3ac1df0072ab39").get(function(err, conference) {
    console.log(conference.status);
});

File: rest/conference/instance-get-example-1/instance-get-example-1.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;
class Example 
{
  static void Main(string[] args) 
  {
    // Find your Account Sid and Auth Token at twilio.com/user/account
    string AccountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    string AuthToken = "your_auth_token";
    var twilio = new TwilioRestClient(AccountSid, AuthToken);

    var conference = twilio.GetConference("CFbbe46ff1274e283f7e3ac1df0072ab39");
    
    Console.WriteLine(conference.Status);
  }
}